bikes/tests.py

```diff
@@ -83,3 +83,45 @@ def test_delete_bike_successful_doesnt_exist(self):
         bike = Bike.objects.create()
         self.client.delete(reverse("bike-detail", kwargs={"pk": bike.id}))
         self.assertFalse(Bike.objects.filter(id=bike.id).exists())
+
+
+class BikesGetRentedTestCase(TestCase):
+    def test_get_rented_bikes_status_code(self):
+        response = self.client.get(reverse("rented-bike-list"))
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+
+    def test_get_bikes_body(self):
+        user = User.objects.create(first_name="John", last_name="Doe")
+        station = Station.objects.create(name="Station Name")
+        Bike.objects.create(user=user, station=station, status=BikeStatus.working)
+        bike1 = Bike.objects.create(
+            user=user, station=station, status=BikeStatus.in_service
+        )
+        bike2 = Bike.objects.create(
```

Comment:
Po co tworzyć dwa obiekty, które są takie same

Reply:
To jest endpoint listy, więc chciałem żeby w assercie było więcej niż jeden element w liście (patrz niżej)

Reply:
No tak, ale czemu dwa razy nie użyjesz bike1 zamiast towrzyć bike2 który jest dokładnie taki sam

Reply:
Muszą być faktycznie dwa rowery w bazie danych, to nie jest tworzenie zmiennych lokalnych, to jest tworzenie faktycznych entries w DB. To że je do czegoś przypisuje, to tylko ułatwia odwołanie się do nich w assert'cie

Reply:
W sumie tworzę 4 rowery, chcę żeby tlyko dwa z nich pojawiły się na liście (bo tylko dwa są wypożyczone), są to różne rowery, bo mają oddzielne wpisy w bazie, ale też nie jest tak jak mówsz, że są identyczne, bo `bike1.id != bike2.id`.